src/unix/solarish/mod.rs

```diff
@@ -61,6 +67,53 @@ impl ::Clone for ucred_t {
     }
 }
 
+#[cfg_attr(feature = "extra_traits", derive(Debug))]
+pub enum lgrp_content_t {
+    LGRP_CONTENT_ALL = 0,
```

Comment:
Why is this the only (new) enum type that is assigned a discriminant? I imagine they should be consistent in this manner?

Reply:
I forgot to remove as I had another plan originally.